lib/core/msdf-sys/build.rs

```diff
@@ -0,0 +1,39 @@
+extern crate download_lp;
+
+mod msdfgen_wasm {
+
+    pub const VERSION  : &str = "v1.0";
+    pub const FILENAME : &str = "msdfgen_wasm.js";
+
+    pub fn download() {
+        let url = format!(
+            "https://github.com/luna/msdf/releases/download/{version}/{filename}",
```

Comment:
I would refactor the address of our github to a variaable as well. I believe that we may want to refer to it from different places, so refactoring it now could be super helpful. Until we need it in other places, lets leave it in this library / module, but as a variable.

Reply:
done